specs/rust/cargo_check.yaml

```diff
@@ -0,0 +1,9 @@
+---
+name: Check for build errors/warnings in a Rust project
+command: cargo check
+tags:
+  - rust
+  - cargo
+  - check
+description: Without building binaries, checks a Rust project according to the Cargo.toml file in this or a parent directory.
+author: Sutton Bradley
```

Comment:
In general, the Warp team tries to approve workflows that are a little more complicated than just vanilla commands like `cargo check` and `cargo clean`

See more information here: https://github.com/warpdotdev/Workflows#what-makes-a-useful-workflow.

Would you mind removing some of these workflows that are just reiterations of the `cargo` man page? The warp completion engine should already be suggesting these commands since they aren't a combination of a bunch of different flags and arguments that are often much harder to remember